components/ContributionConfirmationModal.js

```diff
@@ -49,7 +51,7 @@ const ContributionConfirmationModal = ({ order, onClose }) => {
   const [platformTip, setPlatformTip] = useState(platformTipAmount);
   const [paymentProcessorFee, setPaymentProcessorFee] = useState(0);
   const [hostFeePercent, setHostFeePercent] = useState(defaultHostFeePercent);
-  const [processedAt, setProcessedAt] = useState();
+  const [processedAt, setProcessedAt] = useState(new Date().toISOString().split('T')[0]);
```

Comment:
Where was this syntax first used? Looks a bit weird, I would avoid duplicating everywhere and used a common helper instead.

Reply:
This was first used in AddFundsModal; 

https://github.com/opencollective/opencollective-frontend/blob/bbe63b0c5aa5b61e7ba71be33d031f9db912c931/components/host-dashboard/AddFundsModal.js#L221

I've refactored both places and added a common helper in utils.